CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+# Contributing
+
+## How to Contribute
+
+We welcome contributions in the form of issues or pull requests!
+
+We want this to be a place where all are welcome to discuss and contribute, so please note that this project is released with a Contributor Code of Conduct. By participating in this project you agree to abide by its terms. Find the code of conduct in the ``CONDUCT.md`` file on GitHub.
+
+If you have a problem using Autofocus or see a possible improvement, open an issue in the GitHub issue tracker. Please be as specific as you can.
+
+If you see an open issue you'd like to be fixed, take a stab at it and open a PR!
+
+## Steps for Making a Pull Request
+
+1. Fork the project from GitHub.
+2. Clone the forked repo to your local disk. 
```

Comment:
I think that `disk` is overly formal. What about "clone the forked repo to your local development environment"?

Reply:
Also you could be developing on a remote machine. Changed.